mysqlnd_azure.c

```diff
@@ -340,9 +340,16 @@ MYSQLND_METHOD(mysqlnd_azure_data, connect)(MYSQLND_CONN_DATA ** pconn,
         unsigned int ui_redirect_port = 0;
         zend_bool serverSupportRedirect = get_redirect_info(conn, redirect_host, redirect_user, &ui_redirect_port);
```

Comment:
I am wondering if customer turned OFF the redirect, will code logic below still run?

Reply:
No, there is a check outside this funtion below, in MYSQLND_METHOD(mysqlnd_azure, connect), if (MYSQLND_AZURE_G(enableRedirect) == REDIRECT_OFF) {xx}

Reply:
so it will not run into this function in the first place